dev-support/release-vm/README.md

```diff
@@ -0,0 +1,63 @@
+# HBase Release Env
+
+This is a vagrant project that provides a virtual machine environment suitable
+for running an Apache HBase release.
+
+Requires:
+* VirtualBox
+* Vagrant
+* The private portion of your signing key avilable in the local GPG agent
+* The private portion of your Github authentication key available in the local GPG agent
+
+## Usage
+
+Unlock the local keyring before proceeding (this should prompt you for your GPG passphrase). For
+example, assuming you have an authentication key configured in your keyring, this will do the
+trick.
+
+```sh
+$ ssh -T git@github.com
+Hi <you>! You've successfully authenticated, but GitHub does not provide shell access.
+```
+
+Export the public portion of your signing credentials where the guest VM can access it,
+
+```sh
+$ gpg --export <you>@apache.org > gpg.<you>.apache.pub
```

Comment:
I think whats missing here is that the cwd is ./release-vm so the .pub is in an importable location and the vagrant up just works?

Reply:
You're right, i never explicitly state this command is run from the `dev-support/release-vm` dir, but that working directory, the directory containing the `Vagrantfile` is mounted to `/vagrant` in the guest.

Reply:
Fix this on commit?